e2e/following.js

```diff
@@ -0,0 +1,92 @@
+const assert = require('chai').assert;
+
+const {createWebdriver, E2E_INDEX} = require('./utils');
+const {By, until} = require('selenium-webdriver');
+
+describe('followingAndFollower', () => {
+  const driver = createWebdriver();
+
+  after(() => {
+    driver.quit();
+  });
+
+  it('open index page', async () => {
+    await driver.get(E2E_INDEX);
+  });
+
+  it('register1', async () => {
+    const registerName = await driver.findElement(By.name('registerName'));
+    const registerPassword = await driver.findElement(By.name('registerPassword'));
+
+    await registerName.sendKeys('testName1');
+    await registerPassword.sendKeys('testPassword1');
+    await registerName.submit();
+
+    await driver.wait(until.elementLocated(By.css('.alert')));
+    const alert = await driver.findElement(By.css('.alert'));
+    assert.match(await alert.getText(), /Registered!/);
+  });
+
+  it('register2', async () => {
```

Comment:
Seems like the two callbacks are reusable

Reply:
We've refactored the register part